examples/howto/Model_Builder.ipynb

```diff
@@ -0,0 +1,676 @@
+{
```

Comment:
<p>Misses "." at the end.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/419/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>